js/src/dropdown.js

```diff
@@ -289,6 +289,20 @@ class Dropdown extends BaseComponent {
     return this._element.closest(`.${CLASS_NAME_NAVBAR}`) !== null
   }
 
+  _getOffset() {
```

Comment:
@rohit2sharma95 this seems to be duplicated. Might be worth checking at a later time if we it makes sense to deduplicate this. Lower priority right now :)

Reply:
Yes, should tackle it in a new PR later?

Reply:
I'm trying to deduplicate this into `utils/index.js` and I was wondering if we can do the same for `_getPopperConfig`...
but the function signature becoming a little bit weird:
```js
const getPopperOffsets = (element, config) => {
  if (typeof offset === 'string') {
    return offset.split(',').map(val => Number.parseInt(val, 10))
  }

  if (typeof offset === 'function') {
    return popperData => offset(popperData, element)
  }

  return offset
}

const getPopperConfig = (element, placement, offset, modifiers, options) => {
  const defaultBsConfig = {
    placement: placement,
    modifiers: [...(modifiers || [])],
    ...(options || {})
  }

  if (offset) {
    defaultBsConfig.modifiers.push({
      name: 'offset',
      options: {
        offset: getPopperOffsets(element, offset)
      }
    })
  }

  return defaultBsConfig
}
```
the `element` is necessary because offset configured as function needs it for the second parameter.

Reply:
No need to do this now. We'll have plenty of time after this lands :)